packages/IMAPClient-Core.package/ICEndPoint.class/instance/folderFromResponse..st

```diff
@@ -1,4 +1,4 @@
-protocol parse
+protocol handling
```

Comment:
why is this category not as specific as the others? e.g. protocol handling - move

Reply:
Because this is a row of functions which will be executed together somehow.